static/menu.js

```diff
@@ -171,21 +171,43 @@
                     e.stopPropagation();
                     break;
                 case "home":
-                case "pageup":
                     // home: focus first menu item.
                     // This is the behavior of WAI, while GitHub scrolls,
                     // but it's unlikely that a user will try to scroll the page while the menu is open,
                     // so they won't do it on accident.
                     switchTo = allItems[0];
                     break;
                 case "end":
-                case "pagedown":
                     // end: focus last menu item.
                     // This is the behavior of WAI, while GitHub scrolls,
                     // but it's unlikely that a user will try to scroll the page while the menu is open,
                     // so they won't do it on accident.
                     switchTo = last(allItems);
                     break;
+                case "pageup":
+                    // page up: jump five items up, stopping at the top
+                    // the number 5 is used so that we go one page in the
+                    // inner-scrolled Depedencies and Versions fields
+                    switchTo = currentLink;
+                    for (var n = 0; n < 5 && switchTo; ++n) {
+                        switchTo = previous(allItems, switchTo);
+                    }
```

Comment:
This causes page-up to go _between_ the lists, which is not exactly the behavior I'd expect. e.g. if you have the first dependency highlighted it will behave as if you hit shift+tab 5 times, taking you to 'Github Stars'. I'd expect it to go to the top of the dependency list and stop there, requiring you to hit shift+tab if you want to change focus.

Reply:
Sure, it can be fixed to obey menu sections.